ArduPlane/Plane.h

```diff
@@ -217,6 +217,9 @@ class Plane : public AP_Vehicle {
     // are we doing calibration? This is used to allow heartbeat to
     // external failsafe boards during baro and airspeed calibration
     bool in_calibration;
+    
+    // are we currently in long failsafe>
+    bool long_failsafe_active;
```

Comment:
long_failsafe_pending would be a better name

Reply:
done